pkg/controller/oneagent/istio_test.go

```diff
@@ -129,6 +42,21 @@ func TestReconcileOneAgent_ReconcileIstio(t *testing.T) {
 	}
 }
 
+func TestReconcileOneAgent_ReconcileIstioViaDynatraceClient(t *testing.T) {
```

Comment:
The `TestReconcileOneAgent_ReconcileIstio` test above does everything done in this test as well. Wouldn't that make `TestReconcileOneAgent_ReconcileIstioViaDynatraceClient` superfluous?

Reply:
`TestReconcileOneAgent_ReconcileIstioViaDynatraceClient` might need more work in terms of testing istio endpoints which can either be added in consequent PR.